apm-agent-core/src/main/java/co/elastic/apm/agent/logging/LoggingConfiguration.java

```diff
@@ -176,7 +176,9 @@ static LogLevel mapLogLevel(LogLevel original) {
             " - OVERRIDE - same log output is used, but in ECS-compatible JSON format instead of the original format. \n" +
             "\n" +
             "NOTE: while `SHADE` and `REPLACE` options are only relevant to file log appenders, the `OVERRIDE` option \n" +
-            "is also valid for other appenders, like System out and console")
+            "is also valid for other appenders, like System out and console.\n" +
+            "This option only applies to pattern layouts/formatters by default.\n" +
+            "See also <<config-log-ecs-formatter-allow-list, `log_ecs_formatter_allow_list`>>.")
```

Comment:
This is true for all options, so I would add it to the general description rather than this note